packages/babel-parser/test/fixtures/core/sourcetype-unambiguous/import-meta/options.json

```diff
@@ -1,4 +0,0 @@
-{
-  "sourceType": "unambiguous",
-  "plugins": ["importMeta"]
```

Comment:
This test is still relevant, even without the plugin, because we are testing the script vs module detection.